ambari/src/test/resources/ambari-cluster-with-notifications.yaml

```diff
@@ -0,0 +1,31 @@
+name: Ambari driven cloud - Small
+services:
+- type: io.brooklyn.ambari.AmbariCluster
+  name: Ambari Cluster
+  brooklyn.config:
+    securityGroup: test-ambari
+    ambariAlertNotifications:
+      name: BDaAP alerts #Notification Name
+      description: Ambari Alerts Notification for customer #Notification Description - OPTIONAL
+      notification_type: EMAIL #Notification type - OPTIONAL. If missing will use the default configured in brooklyn.properties
+      alert_states: [OK, WARNING, CRITICAL, UNKNOWN] #Alert states - OPTIONAL. If missing will use the default configured in brooklyn.properties
+      properties:
+        ambari.dispatch.recipients: email.to@example.com, email2@example.com #Notification Recipients
+        mail.smtp.from: email.from@example.com #Mail from - OPTIONAL. If missing will use the default configured in brooklyn.properties
+        mail.smtp.host: examplehost #Mail Host - OPTIONAL. If missing will use the default configured in brooklyn.properties
```

Comment:
It doesn't make checks. From the Ambari server's UI you can also add fake properties and there is only format validation. It will not raise an error until the first time it tries to send notification.